autoload/tig_explorer.vim

```diff
@@ -137,7 +137,7 @@ endfunction
 function! s:tig_callback(exit_code) abort
   if a:exit_code == 0
     if has('nvim')
-      silent! Bclose!
+      silent! Bclose
```

Comment:
On `NeoVim`, Bclose remains read-only buffers after do tig_callback

```vim
:buffers
  3 %a   "autoload/tig_explorer.vim"    line 141
  4 #h-  "term://.//23306:TIGRC_USER=/tmp/nvimSwU80w/3 tig status" line 39
```

Reply:
On SpaceVim, could you work the following workaround instead of `silent! Bclose!`?

```suggestion
      silent! buffer # | bwipeout! #
```